src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,20 @@
+package Duke;
+
+import java.util.Scanner;
+import Duke.Tasks.*;
+import Duke.Exceptions.*;
```

Comment:
I think that specifying the modules imported here might be more concise.

Reply:
Yeah I have removed wildcard imports